### PR TITLE
Feature/styling

### PR DESCRIPTION
**Pull request by ClintonYounge**
**Update**
- Added styling to match Zeplin Template.

Find the template here- https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378

### CODE REVIEW

File: src/styles/book.css [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+
+.progress-stat {
+  margin-right: 40px;
+  font-size: 25px;
+  margin-top: 15px;
+}
+
+.up-progress {
+  margin-top: 15px;
+  background-color: rgb(77, 77, 255);
```
Comment (Tufahel) [inline src/styles/book.css]: - [ ] The `background color` you have used for the buttons are not matching with the given design guideline. Kindly implement the given design guideline for styling. Please implement the color below for the background color of the button.
- image

File: src/styles/book.css [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: white;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  margin: 1.188rem 0 0 2.125rem;
+  border-radius: 3px;
+  background-color: rgb(77, 77, 255);
+}
```
Comment (Tufahel) [inline src/styles/book.css]: - [ ] The `background color` you have used for the buttons are not matching with the given design guideline. Kindly implement the given design guideline for styling. Please implement the color below for the background color of the button.
- image

File: src/styles/header.css [hunk 3]
```diff
@@ -1,42 +1,63 @@
...
 nav {
   width: 90%;
-  height: 100px;
-  display: flex;
   margin: auto;
-  align-items: center;
+  height: 5.5rem;
+  padding: 1.438rem 6.188rem 1.688rem 2.8rem;
+  background-color: var(--white-two);
 }
```
Comment (Tufahel) [inline src/styles/header.css:26]: - ***Your Design***
image

- ***Given Design***
image

- [ ] Color of the navigation menu such as: `Books` and `Categories` are not matching with the given design. From the given design you can see when a menu is `clicked` it becomes `darker` such as `Books`, and when a  menu is `unclicked` it becomes `lighter` such as `Categoies`. Kindly follow the design guideline for the color and use it in your design

File: src/styles/header.css [hunk 4]
```diff
@@ -1,42 +1,63 @@
...
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: rgb(77, 77, 255);
 }
```
Comment (Tufahel) [inline src/styles/header.css]: - [ ] The `color` you have used for the `BookStore CMS` is not matching with the given design guideline. Kindly implement the given design guideline for styling. Please implement the color below for the color of the `Logo`.
- image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @ClintonYounge  :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Descriptive pull request.
✔️ Follow the correct Git flow.
✔️ Professional Readme.
✔️ Application is working fine.

## Required Changes ♻️

***Please kindly check the comments/review below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ClintonYounge** [comment]
@Tufahel Thanks. Implemented all the requested changes.

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @ClintonYounge,

#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

giphy (1)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by ClintonYounge** [comment]
@Omar-Muhamad Incredible. Thanks for the review.

**Pull request merged**